06-wrappers/06-UiInputDate/components/UiInputDate.vue

```diff
@@ -1,13 +1,101 @@
 <template>
-  <UiInput />
+  <UiInput v-model="computedModelValue" :type="type" :step="stepValue" ref="input">
+    <template v-if="$slots['left-icon']" #left-icon>
+      <slot name="left-icon" />
+    </template>
+    <template v-if="$slots['right-icon']" #right-icon>
+      <slot name="right-icon" />
+    </template>
```

Comment:
It's better to proxy all the slots instead of proxy each slot individually. It is shorter by the code (3 lines instead of 6). And if `UiInput` would have a new slot one day or changes in slots, there would be no need to modify `UiInputDate`.